bedrock/firefox/tests/test_helpers.py

```diff
@@ -52,6 +52,7 @@ def check_dumb_button(self, doc):
         eq_(pq(links[5]).attr('href'), settings.APPLE_APPSTORE_FIREFOX_LINK
                                             .replace('/{country}/', '/'))
 
+    @patch('bedrock.base.waffle.switch', Mock(return_value=False))
```

Comment:
> Try mocking switch where it's imported instead of where it's defined:

> @patch('bedrock.firefox.firefox_details.switch', Mock(return_value=False))
def test_button_force_direct(self):
    ...

Hmm, so even this seems to fail still locally :(